test/createEpicMiddleware-spec.js

```diff
@@ -13,16 +13,18 @@ import { mergeStatic } from 'rxjs/operator/merge';
 import { mapTo } from 'rxjs/operator/mapTo';
 
 describe('createEpicMiddleware', () => {
-  it('should provide epics a stream of action$ in and the "lite" store', () => {
+  it('should provide epics a stream of action$ in and the "lite" store', (done) => {
     const reducer = (state = [], action) => state.concat(action);
     const epic = sinon.stub().returns(empty());
     const epicMiddleware = createEpicMiddleware(epic);
     const mockMiddleware = store => next => action => {
-      expect(epic).to.have.been.calledOnce();
+      expect(epic.calledOnce).to.equal(true);
```

Comment:
Ah, the calledOnce should have come from https://github.com/domenic/sinon-chai - that wasn't set up here.